scripts/generate_datafed.sh

```diff
@@ -206,6 +213,8 @@ export DATAFED_WEB_CERT_PATH="$local_DATAFED_WEB_CERT_PATH"
 export DATAFED_WEB_USER=""
 # How the web server communicates with the core server, assumes an internal network
 export DATAFED_CORE_ADDRESS_PORT_INTERNAL="$local_DATAFED_CORE_ADDRESS_PORT_INTERNAL"
+# The id for the associated Google Analytics tag, if left empty, Google Analytics will be disabled
+export DATAFED_GOOGLE_ANALYTICS_TAG="$local_DATAFED_GOOGLE_ANALYTICS_TAG"
```

Comment:
We probably need a better description of where this comes from. Where in google analytics do you get it from? Maybe an example of what one might look like.